meta-hovergames-distro/recipes-gnome/gdk-pixbuf/gdk-pixbuf_2.38.2.bbappend

```diff
@@ -0,0 +1,18 @@
+#
+# This class will generate the proper postinst/postrm scriptlets for pixbuf
+# packages.
+#
+
+
+pixbufcache_common() {
```

Comment:
What's going on in these bbappends? Ubuntu rootfs magic I guess. If you learned something you might comment the files or the commit.

Reply:
So this recipe originally inherits the pixbuf bbclass which causes the the post-install errors when making the demo image. Yocto doesn't have a way to remove an inherit but you can declare them as an empty function which will override the original inherited function.